src/OgRoleManagerInterface.php

```diff
@@ -46,6 +46,19 @@ public function getDefaultRoles();
   public function getRequiredDefaultRoles();
 
   /**
+   * Returns all the roles of a provided group.
+   *
+   * @param string $entity_type_id
+   *    The entity type id of the group.
+   * @param string $bundle
+   *    The bundle of the group.
+   *
+   * @return \Drupal\og\OgRoleInterface[]
+   *   An array of roles indexed by their ids.
+   */
+  public function loadRolesByGroup($entity_type_id, $bundle);
```

Comment:
Yeah that makes sense, let's not force our users to have to call `OgRole::create()` if they have no group entity nearby :D